src/logic/actions/DisplayActions.ts

```diff
@@ -0,0 +1,62 @@
+import {EditorModel} from "../../staticModels/EditorModel";
+import {NumberUtil} from "../../utils/NumberUtil";
+import {DisplaySettings} from "../../settings/DisplaySettings";
+import {ISize} from "../../interfaces/ISize";
+import {IRect} from "../../interfaces/IRect";
+import {ImageUtil} from "../../utils/ImageUtil";
+import {RectUtil} from "../../utils/RectUtil";
+import {IPoint} from "../../interfaces/IPoint";
+import {PointUtil} from "../../utils/PointUtil";
+
+export class DisplayActions {
+    public static calculateViewPortSize(): ISize {
+        if (!!EditorModel.editor) {
+            const editorRect: ClientRect | DOMRect = EditorModel.editor.getBoundingClientRect();
+            return {
+                width: editorRect.width,
+                height: editorRect.height
+            }
+        } else {
+            return null;
+        }
+    }
+
+    public static calculateDefaulRenderImageRect(): IRect {
```

Comment:
jeszcze nie wiem co to znaczy "DefaultRenderImage" :P 
no właśnie... łatwiej by mi było gdybyś miał przygotowany mockup/rysunek mniej więcej taki jak Ci podsyłałem

Reply:
To już omówiliśmy :)